### PR TITLE
Migrate to the new service brew syntax

### PR DESCRIPTION
**Pull request by pakrym-stripe**
For the reference the conversion from new syntax to plist happens in https://github.com/Homebrew/brew/blob/master/Library/Homebrew/service.rb

### CODE REVIEW

File: stripe-mock.rb [hunk 1]
```diff
@@ -47,37 +47,21 @@ def install
 
   plist_options :startup => false
 
-  def plist; <<~EOS
-    <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-<plist version="1.0">
-  <dict>
-    <key>KeepAlive</key>
```
Comment (pakrym-stripe) [inline stripe-mock.rb:55]: `keep_alive successful_exit: false`

File: stripe-mock.rb [hunk 2]
```diff
@@ -47,37 +47,21 @@ def install
...
-    <dict>
-      <key>SuccessfulExit</key>
-      <false/>
-    </dict>
-    <key>Label</key>
```
Comment (pakrym-stripe) [inline stripe-mock.rb:60]: The value we used here is the one that's used automatically

File: stripe-mock.rb [hunk 3]
```diff
@@ -47,37 +47,21 @@ def install
...
-    <string>#{plist_name}</string>
-    <key>ProgramArguments</key>
-    <array>
-      <string>#{opt_bin}/stripe-mock</string>
-      <string>-http-port</string>
-      <string>12111</string>
-      <string>-https-port</string>
-      <string>12112</string>
-    </array>
-    <key>RunAtLoad</key>
```
Comment (pakrym-stripe) [inline stripe-mock.rb:70]: true by default

File: stripe-mock.rb [hunk 4]
```diff
@@ -45,39 +45,18 @@ def install
     end
   end
 
-  plist_options :startup => false
```
Comment (pakrym-stripe) [inline stripe-mock.rb:48]: false is the default

**Review by richardm-stripe — APPROVED** [review]

**Pull request merged**